website/docs/add-secure-apps/flows-stages/stages/user_login/index.md

```diff
@@ -29,55 +28,63 @@ When creating or editing this stage in the UI of the Admin interface, you can se
     - Days
     - Weeks
 
-    All values accept floating-point values.
+All values accept floating-point values.
+
+### Stay signed in offset
 
-- **Stay signed in offset**: When this is set to a higher value than the default _seconds=0_, the user logging in is shown a prompt, allowing the user to choose if their session should be extended or not. The same syntax as for _Session duration_ applies.
+When this is set to a higher value than the default _seconds=0_, the user logging in is shown a prompt, allowing the user to choose if their session should be extended or not. The same syntax as for _Session duration_ applies.
 
     ![](./stay_signed_in.png)
 
-- **Network binding and GeoIP binding**
+### Remember device
+
+If set to a duration above 0, a cookie will be stored for the duration specified which will allow authentik to know whether the user is signing in from a known or unknown (new) device.
+
+If there's an existing authenticated user session for the user with the same IP address, authentik also classifies this as a known device. See [here](../../../../sys-mgmt/events/notification_rule_expression_policies.mdx#trigger-alert-when-user-logs-in-from-unknown-device) for an example of how an alert can be configured for logins from unknown devices.
+
+### Network binding and GeoIP binding
 
-    When configured, all sessions authenticated by this stage will be bound to the selected network and/or GeoIP criteria.
+When configured, all sessions authenticated by this stage will be bound to the selected network and/or GeoIP criteria.
 
-    Sessions that break this binding will be terminated on use. The created [`logout`](../../../../sys-mgmt/events/event-actions#logout) event will contain additional data related to what caused the binding to be broken:
+Sessions that break this binding will be terminated on use. The created [`logout`](../../../../sys-mgmt/events/event-actions#logout) event will contain additional data related to what caused the binding to be broken:
```

Comment:
on use.. use of what, exactly? Use of the binding? Does this mean that if the selected GeoIP binding, for example, is for Continent and Country, but the session is started in the "wrong" continent, then the session is terminated as soon as the binding is hit/used/implemented?

If Yes, let's drop the "in use" and just say "Sessions that break this binding will be terminated."